src/SQL.php

```diff
@@ -15,6 +15,13 @@
 use FluentSql\Functions\SqlFunction;
 
 class SQL {
+
+	/** @const FORCE_NON_NUMERIC_LIMIT whether to force using non numeric LIMIT clause, e.g. a subquery */
```

Comment:
We only use simple numeric values. MySQL does not allow non-numeric `LIMIT` and `OFFSET`. It does, however, allow alternative syntax `LIMIT x, y` which is equivalent to `LIMIT y OFFSET x`. And I was not sure who else might be using this library and whether other engines (as FluentSQL seems not necessarily MySQL-specific) allow functions or subqueries in `LIMIT`. PostgreSQL and Oracle allow only numbers and `LIMIT ALL` which is equivalent to no limit at all. As for SQLite, any scalar expression may be used in the `LIMIT` clause, so long as it evaluates to an integer or a value that can be losslessly converted to an integer.